src/platform/plugins/shared/controls/public/controls/data_controls/range_slider/get_range_slider_control_factory.tsx

```diff
@@ -192,20 +192,22 @@ export const getRangesliderControlFactory = (): DataControlFactory<
         dataControlManager.api.dataViews$,
         dataControlManager.api.fieldName$,
         selections.value$,
+        min$,
+        max$,
       ])
         .pipe(debounceTime(0))
-        .subscribe(([dataViews, fieldName, value]) => {
+        .subscribe(([dataViews, fieldName, value, min, max]) => {
```

Comment:
```suggestion
        .subscribe(([dataViews, fieldName, value]) => {
```

Reply:
We don't anymore. Before I was checking the value against the min and max but removed that check to simplify the logic here.